src/server/httpServer.ts

```diff
@@ -158,7 +158,10 @@ export class HttpServer extends Disposable {
 
 		if (!basePath && (URLPathName == '/' || URLPathName == '')) {
 			const respInfo = this._contentLoader.createNoRootServer();
-			res.writeHead(404);
+			res.writeHead(404, {
+				'Accept-Ranges': 'bytes',
+				'Content-Type': `${respInfo.ContentType}; charset=UTF-8`,

```

Comment:
I'm not too sure why this is needed on the 404 response?

Reply:
![image](https://user-images.githubusercontent.com/53035692/181009314-978608ab-06e4-4437-935f-235f18bc3c16.png)
for relativePath

Reply:
Ahh I see, it's for when you enter a special character and are given a 404.